hazelcast/src/main/java/com/hazelcast/cache/impl/CacheProxyUtil.java

```diff
@@ -236,4 +239,25 @@ public static int getPartitionId(NodeEngine nodeEngine, Data key) {
             }
         }
     }
+
+    public static <K, V> void validateCacheConfig(CacheConfig<K, V> cacheConfig) {
```

Comment:
is it possible to combine this `validateCacheConfig` and `checkCacheConfig` methods? seems some duplication exists.